.github/filters.yml

```diff
@@ -68,3 +69,5 @@ mgmt:
 ci_cd:
   - .github/actions/**
   - .github/workflows/ci_cd.yml
+vale:
+  - .vale
```

Comment:
Does it need its own section? Why not just keep it under "lint"?

Reply:
Totally, I'll move it to `lint` :+1:

Reply:
Ah, actually, I remember now why I had this separated in addition to under `lint`: the `get-changes` CI step uses this filters.yml to determine the groups of changes, and we need to detect changes in `.vale` to build and publish the vale image when the vale configuration changes. This is what makes the changes in `set_matrix_images.py` actually cause the docker image to get built (and published), because that script checks for `vale` in the set of changes:

https://github.com//WordPress/openverse/blob/b6ccc486ebd561f9777ee5128948e6b2e262c77b/automations/python/workflows/set_matrix_images.py#L61

Reply:
I've removed this altogether in favour of a much simpler local-docker image approach. I updated the PR description with details and rationale.